src/index.js

```diff
@@ -1,33 +1,59 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import HeaderComponent from "./components/header-component";
+import CardComponent from "./components/card-component/index.js";
+import teamdata from "./components/card-component/config";
 
-//create functional nested component
-const name = "Third";
-const SecondHeadingComponent = () => {
-  return <h2>Second Heading</h2>;
+const CardContainer = () => {
+  // using for loop only prints first object of teamdata array
+  // for (let i = 0; i <= teamData.length; i++) {
+  //   console.log(i);
```

Comment:
Try to return an array of JSX elements by the for loop

Reply:
yeah i tried that by pushing it to an array and it worked.